src/containers/CreateDoc/index.tsx

```diff
@@ -1,26 +1,120 @@
-import React from 'react';
-import Dropdown from '../../components/Dropdown';
-import documentTypes from './config';
-import { createPdf } from './pdf';
+import React, { useState } from "react";
+import Dropdown from "../../components/Dropdown";
+import documentTypes from "./config";
+import { createPdf } from "./pdf";
+import "./createDoc.css";
+
+import Button from "@material-ui/core/Button";
+import GetAppIcon from "@material-ui/icons/GetApp";
+import VisibilityIcon from "@material-ui/icons/Visibility";
+import createDocStyles from "./styles";
+import DeleteIcon from "@material-ui/icons/Delete";
 
 const CreateDoc: React.FC = () => {
+  const [docProcessInitialized, handleProcessInitialization] = useState<
+    Boolean
+  >(false);
+
+  const classes = createDocStyles();
 
-    function openPdf(): void {
-        createPdf().open();
-    }
+  function openPdf(): void {
+    createPdf().open();
+  }
 
-    function downloadPdf() : void {
-        createPdf().download();
-    }
+  function downloadPdf(): void {
+    createPdf().download();
+  }
 
-    return (
-        <React.Fragment>
-            <h1>CreateDoc Page</h1>
-            <Dropdown name="documentType" title="Document type" options={documentTypes} />
-            <button onClick={openPdf}>See preview</button>
-            <button onClick={downloadPdf}>Download PDF</button>
-        </React.Fragment>
-    );
-}
+  return (
+    <section className="page_main">
+      <div className="page__content">
+        <ul className="Dont-know-what-is-this">
```

Comment:
Do you still need the `Dont-know-what-is-this` class ?

Reply:
Same premonition. Same same. My bad